use snprintf instead of deprecated sprintf.

rpajarola opened the pull request:
Pretty straightforward, avoids nasty warnings on MacOS X.

For some weird reason, this only works when deleting those empty lines in libde265.cc (something getting cached I assume, but I can't for the life of me figure out what).

AndreiMisiukevich commented on README.md:
````diff
@@ -3,7 +3,7 @@
 - `heic2jpg` to convert HEIC files to JPG preserving exif 
 
-``` go get github.com/jdeng/goheif/...```
+``` go get github.com/rpajarola/goheif/...```
````
:) why have you updated all these lines?
I am sure they should be reverted

rpajarola replied:
I didn't mean to include that here. Let me revert that since it will no longer be necessary if you accept the rest of this patch.

rpajarola commented:
sorry, I deleted the last commit, but that also deleted the pull request. I
created a new one.


On Mon, Jun 2, 2025 at 12:00 AM Andrei ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In README.md
> <https://github.com/jdeng/goheif/pull/29#discussion_r2120236149>:
>
> > @@ -3,7 +3,7 @@
>  ## Install
>  - `heic2jpg` to convert HEIC files to JPG preserving exif
>
> -``` go get github.com/jdeng/goheif/...``` <http://github.com/jdeng/goheif/...>
> +``` go get github.com/rpajarola/goheif/...``` <http://github.com/rpajarola/goheif/...>
>
> :) why have you updated all these lines?
> I am sure they should be reverted
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jdeng/goheif/pull/29#pullrequestreview-2887024844>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABXLENSL5INC5EBD4524N2D3BPY7PAVCNFSM6AAAAABWKEQAI2VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDQOBXGAZDIOBUGQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
